config/cloudinary.yml

```diff
@@ -0,0 +1,21 @@
+---
+development:
+  cloud_name: <%= ENV['CLOUD_NAME'] %>
+  api_key: <%= ENV['API_KEY'] %>
+  api_secret: <%= ENV['API_SECRET'] %>
+  enhance_image_tag: true
+  static_file_support: false
+  cdn_subdomain: true
+production:
+  cloud_name: <%= ENV['CLOUD_NAME'] %>
+  api_key: <%= ENV['API_KEY'] %>
+  api_secret: <%= ENV['API_SECRET'] %>
+  enhance_image_tag: true
+  static_file_support: true
+  cdn_subdomain: true
+test:
```

Comment:
I think you could remove this, if you don't change the `active_storage.service` in `environments/test.rb`